### ISSUE TITLE
ApiContext::restore() billing_contract is null resulting in "BunqModel::createFromResponseArray() must be of the type array" error

### ISSUE THREAD

**Issue by RoodFruit** [bunq/sdk_php#214]
## Steps to reproduce:
1. Create a **new** bunq json file as a company (UserCompany) using `ApiContext::create(...`. Using your existing one will not produce the error, since the node `billing_contract` causing it, is probably not in your json file yet.
2. Run a command to the API. This uses your existing ApiContext by using `ApiContext::restore(...`
3. Using this `restore()` will create an error:
```
Argument 1 passed to bunq\Model\Core\BunqModel::createFromResponseArray() must be of the type array, null given, called in ...vendor/bunq/sdk_php/src/Model/Core/BunqModel.php on line 254
```

## What should happen
1. Return an empty array for the field `billing_contract` of UserCompany instead of a null value.

## What happens:
1. In your newly created bunq json file is a node called `billing_contract` like so:
```
"billing_contract": [
     null,
     null,
     null
],
```
Since this is null, the the parser prompts an error resulting in an unusable API.

## SDK version and environment
- Tested on [1.14.18](https://github.com/bunq/sdk_php/releases/tag/1.14.18)
- [X] Production

## Dirty workaround
- Manually remove the `billing_contract` node in your production json and this will work again...

**Comment by RoodFruit** [comment]
See #206

**Comment by RoodFruit** [comment]
Bunq fixed the issue in 1.14.2.
